libs9s/s9srpcclient.cpp

```diff
@@ -11731,3 +11739,9 @@ S9sRpcClient::registerRedisCluster(
 
     return executeRequest(uri, request);
 }
+
+
+bool isValidDateTimeFormat(const std::string& str) {
+    std::regex datetimeRegex("^(\\d{4})-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[01]) (0[0-9]|1[0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])$");
+    return std::regex_match(str, datetimeRegex);
+}
```

Comment:
No empty new line at the end of file, GitHub warns about it for some reason. Not sure if it worth attention. For me it is not a problem because Emacs inserts such line to the end of file automatically on saving.